flink-annotations/src/main/java/org/apache/flink/annotation/docs/Documentation.java

```diff
@@ -41,24 +41,67 @@
 	}
 
 	/**
-	 * Annotation used on config option fields to include them in the "Common Options" section.
+	 * Annotation used on config option fields to include them in specific sections. Sections are groups of options
+	 * that are aggregated across option classes, with each group being placed into a dedicated file.
 	 *
-	 * <p>The {@link CommonOption#position()} argument controls the position in the generated table, with lower values
+	 * <p>The {@link Section#position()} argument controls the position in the generated table, with lower values
 	 * being placed at the top. Fields with the same position are sorted alphabetically by key.
 	 */
 	@Target(ElementType.FIELD)
 	@Retention(RetentionPolicy.RUNTIME)
 	@Internal
-	public @interface CommonOption {
-		int POSITION_MEMORY = 10;
-		int POSITION_PARALLELISM_SLOTS = 20;
-		int POSITION_FAULT_TOLERANCE = 30;
-		int POSITION_HIGH_AVAILABILITY = 40;
-		int POSITION_SECURITY = 50;
+	public @interface Section {
 
+		String SECTION_COMMON = "common";
```

Comment:
`common_section.html` isn't used anymore, but it seems like some classes are still grouped into a generic common section.

Reply:
Actually not, no, this variable is unused. Can remove it.